etc/p3-config/p3-config.yml

```diff
@@ -619,6 +706,232 @@ p3_flavor_storage_B:
+  ram: 6250
+  disk: 12
+  ephemeral: 0
+  extra_specs:
+    "hw_rng:allowed": "True"
+
+octavia_flavor_tiny:
+  name: "amphora.v1.tiny"
+  vcpus: 1
+  ram: 2048
+  disk: 12
+  ephemeral: 0
+  is_public: false
+  extra_specs:
+    "hw_rng:allowed": "True"
+
+p3_flavor_opt_xlarge:
+  name: "optimised.v1.xlarge"
+  vcpus: 30
+  ram: 102400
+  disk: 20
+  ephemeral: 340
+  extra_specs:
+    "trait:HW_CPU_HYPERTHREADING": forbidden
+    "hw_rng:allowed": "True"
+    "hw:mem_page_size": large
+    "hw:numa_nodes": "2"
+    "hw:pci_numa_affinity_policy": required
+    "hw:cpu_policy": dedicated
+    "hw:emulator_threads_policy": share
+    "hw:cpu_sockets": 2
+    "hw:cpu_cores": 15
+    "hw:cpu_threads": 1
+    "hw:cpu_maxsockets": 2
+    "hw:cpu_maxcores": 15
+    "hw:cpu_maxthreads": 1
+
+p3_flavor_opt_large:
+  name: "optimised.v1.large"
+  vcpus: 15
+  ram: 51200
+  disk: 20
+  ephemeral: 160
+  extra_specs:
+    "trait:HW_CPU_HYPERTHREADING": forbidden
+    "hw_rng:allowed": "True"
+    "hw:mem_page_size": large
+    "hw:numa_nodes": "1"
+    "hw:pci_numa_affinity_policy": required
+    "hw:cpu_policy": dedicated
+    "hw:emulator_threads_policy": share
+    "hw:cpu_sockets": 1
+    "hw:cpu_cores": 15
+    "hw:cpu_threads": 1
+    "hw:cpu_maxsockets": 1
+    "hw:cpu_maxcores": 15
+    "hw:cpu_maxthreads": 1
+
+p3_flavor_opt_medium_plus:
+  name: "optimised.v1.medium.plus"
+  vcpus: 8
+  ram: 25600
+  disk: 20
+  ephemeral: 80
+  extra_specs:
+    "trait:HW_CPU_HYPERTHREADING": forbidden
+    "hw_rng:allowed": "True"
+    "hw:mem_page_size": large
+    "hw:numa_nodes": "1"
+    "hw:pci_numa_affinity_policy": required
+    "hw:cpu_policy": dedicated
+    "hw:emulator_threads_policy": share
+    "hw:cpu_sockets": 1
+    "hw:cpu_cores": 8
+    "hw:cpu_threads": 1
+    "hw:cpu_maxsockets": 1
+    "hw:cpu_maxcores": 8
+    "hw:cpu_maxthreads": 1
+
+p3_flavor_opt_medium:
+  name: "optimised.v1.medium"
+  vcpus: 7
+  ram: 25600
+  disk: 20
+  ephemeral: 80
+  extra_specs:
+    "trait:HW_CPU_HYPERTHREADING": forbidden
+    "hw_rng:allowed": "True"
+    "hw:mem_page_size": large
+    "hw:numa_nodes": "1"
+    "hw:pci_numa_affinity_policy": required
+    "hw:cpu_policy": dedicated
+    "hw:emulator_threads_policy": share
+    "hw:cpu_sockets": 1
+    "hw:cpu_cores": 7
+    "hw:cpu_threads": 1
+    "hw:cpu_maxsockets": 1
+    "hw:cpu_maxcores": 7
+    "hw:cpu_maxthreads": 1
+
+p3_flavor_opt_small:
+  name: "optimised.v1.small"
+  vcpus: 3
+  ram: 25600
+  disk: 20
+  ephemeral: 80
+  extra_specs:
+    "trait:HW_CPU_HYPERTHREADING": forbidden
+    "hw_rng:allowed": "True"
+    "hw:mem_page_size": large
+    "hw:numa_nodes": "1"
+    "hw:pci_numa_affinity_policy": required
+    "hw:cpu_policy": dedicated
+    "hw:emulator_threads_policy": share
+    "hw:cpu_sockets": 1
+    "hw:cpu_cores": 3
+    "hw:cpu_threads": 1
+    "hw:cpu_maxsockets": 1
+    "hw:cpu_maxcores": 3
+    "hw:cpu_maxthreads": 1
+
+p3_flavor_opt_small_plus:
+  name: "optimised.v1.small.plus"
```

Comment:
Maybe we drop the plus ones, which would allow the odd miniscule one to plug any gaps?